Sources/WebViewViewController.swift

```diff
@@ -77,6 +77,11 @@ class WebViewViewController: UIViewController, WKScriptMessageHandler {
         }
     }
 
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        self.webview.becomeFirstResponder()
```

Comment:
I remember you have findings on how does this workaround behave on various iOS versions. Please include the findings here for further reference.